Scripts/AssemblyInjections/Unity.Entities/ComponentTypeDependencyExtension.cs

```diff
@@ -10,20 +11,30 @@
 [BurstCompatible]
 public static class ComponentTypeDependencyExtension
 {
-    private static UnsafeList<int> s_WriteTypeList_ScratchPad = new UnsafeList<int>(0, Allocator.Persistent);
-    private static UnsafeList<int> s_ReadTypeList_ScratchPad = new UnsafeList<int>(0, Allocator.Persistent);
+    private static UnsafeList<int> s_WriteTypeList_ScratchPad;
+    private static UnsafeList<int> s_ReadTypeList_ScratchPad;
+
+    [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.SubsystemRegistration)]
+    private static void Init()
+    {
+        s_WriteTypeList_ScratchPad.Dispose();
```

Comment:
I think you have to check if `IsCreated` is called. On MOST collections this will error, but we probably should still have it just in case.

Reply:
OK, fixed